### PR TITLE
Adds Jotaro Kujo's outfit to autodrobe.

### PR DESCRIPTION
**Pull request by Scinexx**
death to mango

nnen9gf

### CODE REVIEW

**Comment by Anderbergst** [comment]
more jojo is always good

**Comment by ArcLumin** [comment]
>no yare yare daze scream change

0/10

**Comment by Spacedong** [comment]
man thats some good shit but please add this ^

**Comment by Scinexx** [comment]
I'm going to do it, give me a bit

**Comment by gumner** [comment]
you have the sprite to the right and the left?

**Comment by Anderbergst** [comment]
it better have ORA ORA ORA ORA ORA ORA ORA ORA ORA ORA ORA ORA ORA

**Comment by yobob591** [comment]
yare yare daze scream pls

**Comment by FuryMcFlurry** [comment]
does this feature inhand sprites.

File: code/game/objects/items/weapons/vending_items.dm [hunk 1]
```diff
@@ -64,7 +64,7 @@
 /obj/item/weapon/vending_refill/autodrobe
 	machine_name = "AutoDrobe"
 	icon_state = "refill_costume"
-	charges = list(29, 4, 3)// of 86 standard, 10 contraband, 9 premium
+	charges = list(31, 4, 3)// of 90 standard, 10 contraband, 9 premium
```
Comment (ghost) [inline code/game/objects/items/weapons/vending_items.dm]: It should be a third rounded up. 90/3 = 30

**Comment by ghost** [comment]
A yare yare alternative scream would be dope.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Looks a little too much like a wizard robe for my taste.

**Comment by Scinexx** [comment]
Now featuring a Yare yare daze and ORAORAORA scream

File: code/game/objects/items/weapons/vending_items.dm (hunk 1 above)
Comment (Scinexx) [reply code/game/objects/items/weapons/vending_items.dm]: fucking math

Comment (Scinexx) [reply code/game/objects/items/weapons/vending_items.dm]: fxied

**Comment by ghost** [comment]
now its got what it needs
+1

**Comment by ghost** [comment]
I'd prefer if the ORA was just a single big ORA. 4 seconds of ORA doesn't suit the scream.

**Comment by Scinexx** [comment]
Yeah, the voice clip I used before was like 10 seconds long, I can change it if you want

**Comment by MethMedic** [comment]
Yare yare....

**Comment by doobiedoo23** [comment]
Can you just replace the scream with dolphin sounds instead?

**Comment by gumner** [comment]
not trying to be the party-ruiner,but you should make each part of clothing separate,aka,you can get shoes,cape,jumpsuit,and hat

**Comment by gumner** [comment]
ehhh nvm i dindt look at the code

**Comment by Spacedong** [comment]
actually lets stop adding more jojo shit this better be the last thing or else its getting out of hand

**Comment by Spacedong** [comment]
also please rename it to delinquent outfit so we don't have a direct reference

**Comment by yobob591** [comment]
dio's costume is called DIO's outfit so

**Comment by Spacedong** [comment]
then rename it to vampire clothes

**Comment by ghost** [comment]
yeah i gotta agree with dong, name it delinquent's clothes or something close to that.

**Comment by trifornt-tinfornt** [comment]
I dunno, given the sprite work I'd say this looks a lot more like Wizard garb.
I demand this be made into wiz robes instead, named delinquent's robes.
Also this better be the last Jojo related thing added to the server. It's getting out of hand.

**Comment by doobiedoo23** [comment]
What's up with the autodrobe creep recently? Everything is in the autodrobe nowadays, and with unoriginal names too. This and DIO's outfit need to be made wizard robes along with the aforementioned name change tbh

**Comment by MethMedic** [comment]
Why does it matter what its called

**Comment by Scinexx** [comment]
I agree with changing he name to make it a little less of an obvious reference, I'll finish when get home

**Comment by SpankMasterson** [comment]
i dont like jojo but the players do +1

**Comment by ArcLumin** [comment]
Code's good. Merging since head approval

**Pull request merged**